Project.toml

```diff
@@ -19,7 +19,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 CEnum = "0.4"
 EnzymeCore = "0.5"
 Enzyme_jll = "0.0.73"
-GPUCompiler = "0.20, 0.21.0"
+GPUCompiler = "0.20.4"
```

Comment:
Why is `0.21` no longer supported?

Reply:
The backport bump of the functionality in GPUCompiler is only in 0.20.x [with LLVM 5 support], or an LLVM 6 0.21.x